src/customTheme.ts

```diff
@@ -109,18 +73,33 @@ const themeConfig: ThemeConfig = {
 const customTheme = extendTheme(
   {
     colors: {
-      blue: blueColor,
-      primary: primaryColor,
-      secondary: blueColor,
-      coolGray,
-      fuchsia,
-      sky,
-      primaryTextLightmode,
-      secondaryTextLightmode,
-      tertiaryTextLightmode,
-      primaryTextDarkmode,
-      secondaryTextDarkmode,
-      tertiaryTextDarkmode,
+      primary: fuchsia,
+      secondary: sky,
+      gray: coolGray,
+    },
+    semanticTokens: {
+      colors: {
+        primaryText: {
+          default: primaryTextLightMode,
+          _dark: primaryTextDarkMode,
+        },
+        secondaryText: {
+          default: secondaryTextLightMode,
+          _dark: secondaryTextDarkMode,
+        },
+        tertiaryText: {
+          default: tertiaryTextLightMode,
+          _dark: tertiaryTextDarkMode,
+        },
+        primaryBg: {
+          default: 'gray.50',
+          _dark: 'gray.900',
+        },
+        secondaryBg: {
+          default: 'gray.100',
+          _dark: 'gray.800',
+        },
+      },
```

Comment:
The [new semantic tokens introduced in Chakra 1.8.1](https://chakra-ui.com/docs/features/semantic-tokens) are really helpful!

Reply:
Oh wow this is really cool! You can now just use `primaryText` and not have to worry about whether it's used in the light or dark mode!

Reply:
Exactly! I hace replaced all the text colors usages. We can extract more things into semantic tokens if needed in the future.